TAF_EasyRest/TestFramework/Pages/Owner/CreateMenuPage.cs

```diff
@@ -0,0 +1,49 @@
+﻿using TestFramework.PageComponents;
+using TestFramework.PageComponents.NavigationMenuComponents;
+using TestFramework.PageComponents.Owner.CreateMenuPageComponents;
+
+namespace TestFramework.Pages.Owner
+{
+    public class CreateMenuPage : BasePage
+    {
+        public NavigationMenuPageComponent NavigationMenuPageComponent { get; }
+        public ManageRestaurantPageComponent ManageRestaurantPageComponent { get; }
```

Comment:
you probably need to add " UserMenuHeaderButtonPageComponent" here as we use this page for user who has signed in already